src/main/java/ch/naviqore/service/impl/walkcalculator/BeeLineWalkCalculator.java

```diff
@@ -0,0 +1,35 @@
+package ch.naviqore.service.impl.walkcalculator;
+
+import ch.naviqore.utils.spatial.GeoCoordinate;
+
+/**
+ * Calculates the walk duration between two points.
+ */
+public class BeeLineWalkCalculator implements WalkCalculator {
+
+    private static final int SECONDS_IN_HOUR = 3600;
+    private final int walkSpeed;
+
+    /**
+     * Creates a new BeeLineWalkCalculator with the given walking speed.
+     *
+     * @param walkSpeed Walking speed in m/h.
```

Comment:
I would go for m/s since this is the SI standard.

Reply:
See my comment in the pull request for the refactorings NAV-87